runtest.py

```diff
@@ -53,9 +54,10 @@ def kill_legacy_process(self):
     def get_test_env(self, **kwargs):
         result = os.environ.copy()
         result['PROFILER'] = self.profiler
-        result['KEEP_BROWSER'] = str(self.keep_browser)
-        result['ENABLE_ONLINE'] = str(self.online)
+        result['KEEP_BROWSER'] = str(int(self.keep_browser))
+        result['ENABLE_ONLINE'] = str(int(self.online))
```

Comment:
hmm, I think we should do something for `--keep-browser` and `--online` switches.
let's modify them in the future refactoring work :)